portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase

```diff
@@ -1107,6 +1107,577 @@ definition {
+    @priority = 5
+    test CanViewWithoutFilters {
+        task ("Given that the user added a Pickilist") {	
+            Picklist.addPicklistViaAPI(picklistName = "Transport");
+
+            Picklist.addPicklistItemViaAPI(
+                itemKey = "bike",
+                itemName = "bike",
+                picklistName = "Transport");
+
+            Picklist.addPicklistItemViaAPI(
+                itemKey = "car",
+                itemName = "car",
+                picklistName = "Transport");
+
+            Picklist.addPicklistItemViaAPI(
+                itemKey = "bus",
+                itemName = "bus",
+                picklistName = "Transport");
+
+            ObjectAdmin.openObjectAdmin();
+
+            ObjectAdmin.addObjectViaAPI(
+                labelName = "Transport",	
+                objectName = "Transport",
+                pluralLabelName = "Transport");
+
+            Refresh();
+
+            ACUtils.waitForScrollDown();
+
+            Click(
+                key_labelName = "Transport",
+                locator1 = "CreateObject#OBJECT_NAME");
+
+            Button.clickSave();
+
+            ObjectAdmin.goToFieldsTab();
+
+            Refresh();
+
+		 }
+
+		task("And Adding items to the created picklist"){
+            LexiconEntry.gotoAdd();
+
+            Type(
+                locator1 = "ObjectField#LABEL_INPUT",
+                value1 = "Filter");
+
+            Click(
+                key_sorting = "",
+                locator1 = "ObjectCustomViews#EDIT_SORTING");
+
+            Click(
+                key_fieldType = "Picklist",
+                locator1 = "ObjectField#TYPE_OPTION");
+
+            Click(locator1 = "ObjectCustomViews#SELECT_AUTO_COMPLETE");
+
+            Click(
+                key_optionName = "Transport",
+                locator1 = "ObjectAdmin#LAYOUT_ADD_FIELD_OPTION");
+
+            Button.clickSave();
+
+            ObjectCustomValidation.gotoTab(tabName = "Details");
+            Button.clickPublish();
+
+			}
+       
+        task ("When the user creates a new data set") {
+            Navigator.openURL();
+
+            DataSetAdmin.goToDataSetAdminPage();
+
+            DataSetAdmin.createDataSetEntryViaAPI(
+                dataSetName = "Dataset Test",
+                restApplication = "/data-set-manager/fields",
+                restEndpoint = "/",
+                restSchema = "FDSField");
+	
+        }
+
+        task ("And a data set view created via api") {
+            DataSetAdmin.createFDSViewViaAPI(
+                dataSetName = "Dataset Test",
+                dataSetViewDescription = "FDSViewDescription",
+                key_dataSetViewNameList = "View Fields Test");
+
+            DataSetAdmin.goToSpecificViewPage(
+                dataSetName = "Dataset Test",
+                key_viewName = "View Fields Test");
+        }
+
+        task ("When goes to Fields Tab.") {
```

Comment:
This should say Filters tab and also you are doing more things than just going to the Filters tab, you are creating the filter itself